### PR TITLE
Correct the Localhost links to be not clickable

### PR DESCRIPTION
**Pull request by chamindi-a**

### CODE REVIEW

File: en/docs/install-and-setup/setup/mi-setup/brokers/configure-with-weblogic.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ This section describes how to configure WSO2 Micro Integrator to connect with Or
 ## Configuring the WebLogic server
 
 Configure the required connection factories and queues in WebLogic. An entry for a JMS queue would look like the following. The configuration
-files can be found inside the `WEBLOGIC_HOME/user_projects/domains/<DOMAIN_NAME>/config/jms` file. Alternatively you can configure using the WebLogic web console, which can be accessed through [http://localhost:7001](http://localhost:7001/) with default configurations.
+files can be found inside the `WEBLOGIC_HOME/user_projects/domains/<DOMAIN_NAME>/config/jms` file. Alternatively you can configure using the WebLogic web console, which can be accessed through `http://localhost:7001`with default configurations.
```
Comment (npamudika) [inline en/docs/install-and-setup/setup/mi-setup/brokers/configure-with-weblogic.md:15]: Missing a space after `http://localhost:7001`

File: en/docs/integrate/develop/customizations/creating-custom-task-scheduling.md [hunk 2]
```diff
@@ -353,11 +353,11 @@ Follow the steps below to create the task and schedule it.
     ```xml
     <task class="org.apache.synapse.startup.tasks.MessageInjector" group="synapse.simple.quartz" name="PrintStockQuoteScheduledTask" xmlns="http://ws.apache.org/ns/synapse">
                 <trigger count="1" interval="5" />
-                <property name="to" value="http://localhost:9000/soap/SimpleStockQuoteService" xmlns:task="http://www.wso2.org/products/wso2commons/tasks" />
+                <property name="to" value= http://localhost:9000/soap/SimpleStockQuoteService />
```
Comment (npamudika) [inline en/docs/integrate/develop/customizations/creating-custom-task-scheduling.md:356]: Don't we need quotation marks here?

File: en/docs/reference/customize-product/extending-api-manager/extending-workflows/invoking-the-api-manager-from-the-bpel-engine.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Invoking the API Manager from the BPEL Engine
+https://localhost:8243/services/WorkflowCallbackService# Invoking the API Manager from the BPEL Engine
```
Comment (npamudika) [inline en/docs/reference/customize-product/extending-api-manager/extending-workflows/invoking-the-api-manager-from-the-bpel-engine.md:1]: Is there any reason to add a URL to the heading?

File: en/docs/install-and-setup/setup/mi-setup/brokers/configure-with-weblogic.md (hunk 1 above)
Comment (chamindi-a) [reply en/docs/install-and-setup/setup/mi-setup/brokers/configure-with-weblogic.md]: Fixed [1aa1beb](https://github.com/wso2/docs-apim/pull/7282/commits/1aa1beb0e2b8ffe48563aded26dcb628c8b4030a)

File: en/docs/integrate/develop/customizations/creating-custom-task-scheduling.md (hunk 2 above)
Comment (chamindi-a) [reply en/docs/integrate/develop/customizations/creating-custom-task-scheduling.md]: Fixed [1aa1beb](https://github.com/wso2/docs-apim/pull/7282/commits/1aa1beb0e2b8ffe48563aded26dcb628c8b4030a)

File: en/docs/reference/customize-product/extending-api-manager/extending-workflows/invoking-the-api-manager-from-the-bpel-engine.md (hunk 3 above)
Comment (chamindi-a) [reply en/docs/reference/customize-product/extending-api-manager/extending-workflows/invoking-the-api-manager-from-the-bpel-engine.md]: Fixed [1aa1beb](https://github.com/wso2/docs-apim/pull/7282/commits/1aa1beb0e2b8ffe48563aded26dcb628c8b4030a)

**Review by npamudika — APPROVED** [review]

**Pull request merged**